config/development-local.js

```diff
@@ -1,4 +1,6 @@
-import { staticPath } from './lib/shared';
+const webpackServerHost = process.env.WEBPACK_SERVER_HOST || '127.0.0.1';
+const webpackServerPort = 7001;
+const webpackDevServer = `${webpackServerHost}:${webpackServerPort}`;
```

Comment:
why do we define these up here, separately, rather than directly within `module.exports`? default.js (or stage.js, etc) don't do it this way.

Reply:
because it is used in two places in the exported config (first as a config var, then as part of another config var)